web/client/configs/localConfig.json

```diff
@@ -104,48 +104,129 @@
             },
             "staticServices": {
               "default_map_backgrounds": {
-                "type": "backgrounds",
-                "title": "Default Backgrounds",
-                "titleMsgId": "defaultMapBackgroundsServiceTitle",
-                "autoload": true,
-                "backgrounds": [{
-                    "type": "osm",
-                    "title": "Open Street Map",
-                    "name": "mapnik",
-                    "source": "osm",
-                    "group": "background"
-                }, {
-                    "type": "tileprovider",
-                    "title": "NASAGIBS Night 2012",
-                    "provider": "NASAGIBS.ViirsEarthAtNight2012",
-                    "name": "Night2012",
-                    "source": "nasagibs",
-                    "group": "background"
-                }, {
-                    "type": "tileprovider",
-                    "title": "OpenTopoMap",
-                    "provider": "OpenTopoMap",
-                    "name": "OpenTopoMap",
-                    "source": "OpenTopoMap",
-                    "group": "background"
-                }, {
-                    "format": "image/jpeg",
-                    "group": "background",
-                    "name": "s2cloudless:s2cloudless",
-                    "opacity": 1,
-                    "title": "Sentinel 2 Cloudless",
-                    "type": "wms",
-                    "url": "https://1maps.geo-solutions.it/geoserver/wms",
-                    "source": "s2cloudless",
-                    "singleTile": false
-                }, {
-                    "source": "ol",
-                    "group": "background",
-                    "title": "Empty Background",
-                    "fixed": true,
-                    "type": "empty"
-                }]
-              }
+				"type": "backgrounds",
```

Comment:
this is different from the original  PR

![image](https://github.com/geosolutions-it/MapStore2/assets/11991428/c1271ce8-6677-437a-9b8e-f2a6b862c5ba)


please fix it